### PR TITLE
[Bots] Wordpress Loopback error page

### PR DESCRIPTION
**Pull request by patriciasantaana**
PCX-4452

### CODE REVIEW

File: content/bots/reference/wordpress-loopback-issue.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Cloudflare has observed that when users attempt to run diagnostics in the Site Status page for Wordpress installations, loopback issues arise when our bot detection services block them. In Wordpress 6.0.1 with no additional plugins or themes, the page generated the following 3 requests:
+
+| Error | Method | URI | User-Agent |
+| ---- | ---- | ---- | ---- |
+| The REST API encountered an unexpected result | `GET` | /wp-json/wp/v2/types/post | WordPress/6.0.1 < site-hostname > |
+| Your site could not complete a loopback request | `GET` | / | WordPress/6.0.1 < site-hostname > |
+| Your site could not complete a loopback request | `POST` | /wp-cron.php | WordPress/6.0.1 < site-hostname > |
+
+These requests all come from the origin IP and not a Cloud service.   As a result, Cloudflare does not have an easy way to add these types of requests to verified bots.
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
These requests all come from the origin IP and not a Cloud service. As a result, Cloudflare does not have an easy way to add these types of requests to verified bots. 
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+These requests all come from the origin IP and not a Cloud service.   As a result, Cloudflare does not have an easy way to add these types of requests to verified bots. 
+
+When these requests are blocked, Wordpress’s block editor screen, Cron events, and automated tests that third-party themes and plugins use cannot function.
+
+Many different Wordpress components perform different loopback tests beyond the original one, making it a complex problem to create exceptions for. 
+
+### Resolution
+
+Enterprise customers can write an allow rule for their origin IP to fix the issue.
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md]: I'd link to relevant Firewall documentation for creating a rule (or an allow rule specifically)

File: content/bots/reference/wordpress-loopback-issue.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+pcx_content_type: reference
+title: Wordpress Loopback error
+weight: 0
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md:4]: weight: 0? Does it still show up in the side navigation?

**Review by kodster28 — APPROVED** [review]

File: content/bots/reference/wordpress-loopback-issue.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+---
+# Wordpress Loopback error
+
+Cloudflare has observed that when users attempt to run diagnostics in the Site Status page for Wordpress installations, loopback issues arise when our bot detection services block them. In Wordpress 6.0.1 with no additional plugins or themes, the page generated the following 3 requests:
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
Cloudflare has observed that when users attempt to run diagnostics in the Site Status page for Wordpress installations, loopback issues arise when our bot detection services block them. 

In **Wordpress 6.0.1** with no additional plugins or themes, the page generated the following 3 requests:
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+| Error | Method | URI | User-Agent |
+| ---- | ---- | ---- | ---- |
+| The REST API encountered an unexpected result | `GET` | /wp-json/wp/v2/types/post | WordPress/6.0.1 < site-hostname > |
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
| The REST API encountered an unexpected result | `GET` | `/wp-json/wp/v2/types/post` | WordPress/6.0.1 < site-hostname > |
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+| Your site could not complete a loopback request | `GET` | / | WordPress/6.0.1 < site-hostname > |
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
| Your site could not complete a loopback request | `GET` | `/` | WordPress/6.0.1 < site-hostname > |
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+| Your site could not complete a loopback request | `POST` | /wp-cron.php | WordPress/6.0.1 < site-hostname > |
```
Comment (kodster28) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
| Your site could not complete a loopback request | `POST` | `/wp-cron.php` | WordPress/6.0.1 < site-hostname > |
```

**Review by kodster28 — APPROVED** [review]

File: content/bots/reference/wordpress-loopback-issue.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+---
+pcx_content_type: reference
+title: Wordpress Loopback error
+weight: 0
+---
+# Wordpress Loopback error
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
# WordPress Loopback error
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
+---
+pcx_content_type: reference
+title: Wordpress Loopback error
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
title: WordPress Loopback error
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+weight: 0
+---
+# Wordpress Loopback error
+
+Cloudflare has observed that when users attempt to run diagnostics in the Site Status page for Wordpress installations, loopback issues arise when our bot detection services block them.
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
Cloudflare has observed that when users attempt to run diagnostics in the Site Status page for WordPress installations, loopback issues arise when our bot detection services block them. 
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
+---
+pcx_content_type: reference
+title: Wordpress Loopback error
+weight: 0
+---
+# Wordpress Loopback error
+
+Cloudflare has observed that when users attempt to run diagnostics in the Site Status page for Wordpress installations, loopback issues arise when our bot detection services block them. 
+
+In **Wordpress 6.0.1** with no additional plugins or themes, the page generated the following 3 requests:
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
In **WordPress 6.0.1** with no additional plugins or themes, the page generated the following 3 requests:
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+| Error | Method | URI | User-Agent |
+| ---- | ---- | ---- | ---- |
+| The REST API encountered an unexpected result | `GET` | `/wp-json/wp/v2/types/post` | WordPress/6.0.1 < site-hostname > |
+| Your site could not complete a loopback request | `GET` | `/` | WordPress/6.0.1 < site-hostname > |
+| Your site could not complete a loopback request | `POST` | `/wp-cron.php` | WordPress/6.0.1 < site-hostname > |
+
+These requests all come from the origin IP and not a Cloud service.  As a result, Cloudflare does not have an easy way to add these types of requests to verified bots. 
+
+When these requests are blocked, Wordpress’s block editor screen, Cron events, and automated tests that third-party themes and plugins use cannot function.
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
When these requests are blocked, WordPress’s block editor screen, Cron events, and automated tests that third-party themes and plugins use cannot function.
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+Many different Wordpress components perform different loopback tests beyond the original one, making it a complex problem to create exceptions for.
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
Many different WordPress components perform different loopback tests beyond the original one, making it a complex problem to create exceptions for. 
```

File: content/bots/reference/wordpress-loopback-issue.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+
+When these requests are blocked, Wordpress’s block editor screen, Cron events, and automated tests that third-party themes and plugins use cannot function.
+
+Many different Wordpress components perform different loopback tests beyond the original one, making it a complex problem to create exceptions for. 
+
+### Resolution
+
+Enterprise customers can [write an allow rule](/firewall/cf-dashboard/create-edit-delete-rules/#create-a-firewall-rule) for their origin IP to fix the issue. 
+
+Self-serve customers can turn off Super Bot Fight Mode or edit `/etc/hosts` file to have loopback requests resolve locally. Refer to this guide on [how to fix Wordpress Loopback errors](https://codeopolis.com/posts/wordpress-loopback-error/).
```
Comment (patriciasantaana) [inline content/bots/reference/wordpress-loopback-issue.md]:
```suggestion
Self-serve customers can turn off Super Bot Fight Mode or edit `/etc/hosts` file to have loopback requests resolve locally. Refer to this guide on [how to fix WordPress Loopback errors](https://codeopolis.com/posts/wordpress-loopback-error/). 
```

**Pull request merged**